en/docs/how-tos/endpoints/overriding-endpoint-information/overriding-endpoints-for-imported-apis.md

```diff
@@ -3,237 +3,115 @@
 !!! note
     This information is only applicable for APIs imported from WSO2 API Manager.
 
-In the event you change your existing backend service connection URL or add a new backend, you will need to override your backend connection URL that you provided in the respective open API definition. The following section describes how you can override your endpoint information for APIs that are imported from WSO2 API Manager.
+In the event of adding a new backend or changing the existing backend service connection URL, you will need to override your backend connection URL that you provided in the respective open API definition. The following section describes how you can override your endpoint information for APIs that are imported from the WSO2 API Manager.
 
-## Overriding a single endpoint
+#### Overriding endpoints at runtime
 
-#### Overriding a single endpoint at runtime
+Use the following command to provide the overriding information at server start-up to override endpoints of an API, which was imported from the WSO2 API Manager.
 
-Use the following command to provide the overriding information at server start-up to override the endpoint of an API, which was imported from WSO2 API Manager.
+Follow the instructions below to override a single endpoint or multiple endpoints, which are either production or sandbox endpoints:
```

Comment:
Follow the instructions below to override the production or sandbox endpoint(s).